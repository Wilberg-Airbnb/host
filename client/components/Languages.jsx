import React from 'react';
import { Item } from 'semantic-ui-react';

const Languages = ({languages}) => (
  <Item>
    <Item.Content>
      <Item.Description>
        Languages: {languages.join(', ')}
      </Item.Description>
    </Item.Content>
  </Item>
)

export default Languages;