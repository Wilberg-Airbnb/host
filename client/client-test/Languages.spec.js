import React from 'react';
import Languages from '../components/Languages.jsx';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
 
configure({ adapter: new Adapter() });
 
describe('Renders Components', () => {
  let component;

  beforeEach( () => {
    component = shallow(<Languages languages={['English', 'Spanish']}/>)
  });

  test("should render initial layout", () => {
    const component = shallow(<Languages languages={['English', 'Spanish']} />);
    expect(component.getElements()).toMatchSnapshot();
  });

});


