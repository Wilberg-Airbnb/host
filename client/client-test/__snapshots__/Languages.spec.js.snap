// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Components should render initial layout 1`] = `
Array [
  <Item>
    <ItemContent>
      <ItemDescription>
        Languages: 
        English, Spanish
      </ItemDescription>
    </ItemContent>
  </Item>,
]
`;
